app/src/panels/time-series/time-series.vue

```diff
@@ -107,6 +107,21 @@ export default defineComponent({
 		const chartEl = ref();
 		const chart = ref<ApexCharts>();
 
+		const yAxisRange = computed(() => {
+			let min = isNil(props.min) ? undefined : Number(props.min);
+			let max = isNil(props.max) ? undefined : Number(props.max);
+
+			if (max && !min) {
+				min = 0;
+			}
+
+			if (max && min && max < min) {
+				max = min;
+				min = Number(props.max);
+			}
+			return { max, min };
```

Comment:
And lets add a return in between here for code style beauty reasons 👨🏻‍🎨

Reply:
Where?

Reply:
Like line break?

Reply:
Yup!

```suggestion
			}

			return { max, min };
```